internal/nodes/ingress_model_rel.go

```diff
@@ -315,6 +316,18 @@ func IngressChanges(ingName string, namespace string, key string) ([]string, boo
 			objects.SharedSvcLister().IngressMappings(metav1.NamespaceAll).RemoveIngressClassMappings(namespace + "/" + ingName)
 		}
 
+		// If the Ingress Class is not found or is not valid, then return.
+		// When the correct Ingress Class is added, then the Ingress would be processed again.
+		if !validateIngressForClass(key, ingObj) {
```

Comment:
This case probably gets covered as part of the L2 translator code.
https://github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/blob/d1b6568c9a1caa4e5273ac5a8e5e2828463d7561/internal/nodes/avi_model_l7_translator.go#L54
and
https://github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/blob/d1b6568c9a1caa4e5273ac5a8e5e2828463d7561/internal/nodes/avi_model_routeingr_hostname_shard.go#L140
where the relevant objects corresponding to an invalid ingress gets deleted (or are not created), if the ingressclass is found to be invalid.